src/functions/template-tags/date.php

```diff
@@ -106,24 +106,34 @@ function tribe_event_is_on_date( $date = null, $event = null ) {
 
 if ( ! function_exists( 'tribe_events_timezone_choice' ) ) {
 	/**
-	 * Event-specific wrapper for wp_timezone_choice().
+	 * Generates a timezone selection dropdown.
+	 *
+	 * This function provides an opportunity to modify the timezone `<option>` elements used within the timezone picker.
+	 * It utilizes the `wp_timezone_choice` function to generate the timezone options and allows for further customization
+	 * through the `tribe_events_timezone_choice` filter.
 	 *
 	 * @since 4.6.5
 	 *
-	 * @param string $selected_zone
-	 * @param string $locale (optional)
+	 * @param string      $selected_zone The selected timezone.
+	 * @param string|null $locale Optional. The locale to use for the timezone choice. Default null.
```

Comment:
Description text should line up, too.
```suggestion
	 * @param string      $selected_zone The selected timezone.
	 * @param string|null $locale        Optional. The locale to use for the timezone choice. Default null.
```